Src/library/net/network/socket/network_socket.e

```diff
@@ -141,7 +141,7 @@ feature -- Status report
 		local
 			retval: INTEGER
 		do
-			retval := c_select_poll_with_timeout (descriptor, True, timeout)
+			retval := c_select_poll_with_timeout (descriptor, True, timeout_ns)
```

Comment:
What about removing the temporary local:
   `Result := c_select_poll_with_timeout (descriptor, True, timeout_ns) > 0`
?

Reply:
ok ... even if I find it easier when debugging to have such retval local variable...